app/models/domain_block.rb

```diff
@@ -108,4 +109,10 @@ def public_domain
   def domain_digest
     Digest::SHA256.hexdigest(domain)
   end
+
+  def severity=(value)
+    super
+  rescue ArgumentError
```

Comment:
Rails, by default, raises an `ArgumentError` instead of validating the values of an enum. I found this workaround [here](https://github.com/rails/rails/pull/41730#issue-838110510), but I'm not sure if it's an acceptable solution.